### DISCUSSION TITLE
Question about setting up providers

### DISCUSSION THREAD

**Discussion by ZiadSenior** [vosmiic/jellyfin-ani-sync#136] [Q&A]
So say I logged into Anilist and made an API Client, do I have to do this once and then connect all of my Jellyfin users with it, or do each need to provide their API Client?

I just want to make sure, thanks in advance :D

**Comment by vosmiic** [comment]
Yeah its just the one API client, so only you need to create one. The other users are able to auth through your API client.
You will want to create an API client, fill in the deatils in the plugin, select the user you want to auth, generate the auth URL and then give it to the user owner. You will want to make sure they are able to reach the server (so on your local network if your server is not reachable online) when they attempt to authenticate with the API.

Hopefully this has clarified how to setup the plugin, do feel free create more discussions if you have any more questions.
